samples/instance-with-autoscaling-config.js

```diff
@@ -0,0 +1,114 @@
+  const spanner = new Spanner({
+    projectId: projectId,
+  });
+
+  // Get the instance admin client
+  const instanceAdminClient = spanner.getInstanceAdminClient();
+
+  const autoscalingConfig =
+    protos.google.spanner.admin.instance.v1.AutoscalingConfig.create({
+      // Only one of minNodes/maxNodes or minProcessingUnits/maxProcessingUnits
+      // can be set. Both min and max need to be set and
+      // maxNodes/maxProcessingUnits can be at most 10X of
+      // minNodes/minProcessingUnits.
+      autoscalingLimits:
+        protos.google.spanner.admin.instance.v1.AutoscalingConfig.AutoscalingLimits.create(
+          {
+            minNodes: 1,
+            maxNodes: 2,
+          }
+        ),
+      // highPriorityCpuUtilizationPercent and storageUtilizationPercent are both
+      // percentages and must lie between 0 and 100.
+      autoscalingTargets:
+        protos.google.spanner.admin.instance.v1.AutoscalingConfig.AutoscalingTargets.create(
+          {
+            highPriorityCpuUtilizationPercent: 65,
+            storageUtilizationPercent: 95,
+          }
+        ),
+    });
+
+  // Creates a new instance
+  try {
+    console.log(
+      `Creating instance ${instanceAdminClient.instancePath(
+        projectId,
+        instanceId
+      )}.`
+    );
+    const [operation] = await instanceAdminClient.createInstance({
+      instanceId: instanceId,
+      parent: instanceAdminClient.projectPath(projectId),
+      instance: {
+        config: instanceAdminClient.instanceConfigPath(
+          projectId,
+          'regional-us-central1'
+        ),
+        displayName: 'Display name for the instance.',
+        autoscalingConfig: autoscalingConfig,
+        labels: {
+          cloud_spanner_samples: 'true',
+          created: Math.round(Date.now() / 1000).toString(), // current time
+        },
+      },
+    });
+
+    console.log(`Waiting for operation on ${instanceId} to complete...`);
+    await operation.promise();
+    console.log(`Created instance ${instanceId}.`);
+
+    // get instance metadata
+    const [metadata] = await instanceAdminClient.getInstance({
+      name: instanceAdminClient.instancePath(projectId, instanceId),
+    });
+    console.log(
+      `Autoscaling configurations of ${instanceId} are:  ` +
+        '\n' +
+        `Min nodes: ${metadata.autoscalingConfig.autoscalingLimits.minNodes} ` +
+        'nodes.' +
+        '\n' +
+        `Max nodes: ${metadata.autoscalingConfig.autoscalingLimits.maxNodes}` +
+        ' nodes.' +
+        '\n' +
+        `High priority cpu utilization percent: ${metadata.autoscalingConfig.autoscalingTargets.highPriorityCpuUtilizationPercent}.` +
+        '\n' +
+        `Storage utilization percent: ${metadata.autoscalingConfig.autoscalingTargets.storageUtilizationPercent}.`
+    );
+  } catch (err) {
+    console.error('ERROR:', err);
+  }
+  // [END spanner_create_instance_with_autoscaling_config]
+}
+
+module.exports.createInstanceWithAutoscalingConfig =
```

Comment:
Follow the rubric syntax . Sample https://github.com/googleapis/nodejs-spanner/blob/main/samples/pg-database-create.js